docs/getting-started.md

```diff
@@ -232,7 +225,682 @@ curl localhost:10101/index/repository/query \
 Please note that while user ID 99999 may not be sequential with the other column IDs, it is still a relatively low number. 
 Don't try to use arbitrary 64-bit integers as column or row IDs in Pilosa - this will lead to problems such as poor performance and out of memory errors.
 
+#### Using Go
+
+Pilosa requires Go 1.12 or higher.
+
+##### Create the Environment
+
+In order to communicate with Pilosa through your Go code, you must have a client, which is go-pilosa. To install go-pilosa, open a terminal (one other than the one running Pilosa) and download the library to your `GOPATH` using:
```

Comment:
Interacting with Pilosa in your go program is best accomplished using our client, go-pilosa. Install go-pilosa (in a new terminal) and download ...